libyul/optimiser/Metrics.cpp

```diff
@@ -48,7 +48,7 @@ size_t CodeSize::codeSize(Expression const& _expression)
 size_t CodeSize::codeSize(Block const& _block)
 {
 	CodeSize cs;
-	cs.visit(_block);
+	cs.visit(Statement{_block});
```

Comment:
We should *not* keep it like this, since this will copy the whole block - but let's see whether the AppVeyor run works like this before fixing it further.

Reply:
Actually the copy happens on develop as well - we should avoid it anyways...

Reply:
Actually I'll fix it right away :-).